src/app/containers/DocumentControl/index.tsx

```diff
@@ -40,24 +59,34 @@ export const DocumentControl = () => {
       </div>
 
       <div className={cx("documentList")}>
-        {documentsBySubject?.documents?.map(document => (
+        {documentsBySubject?.data?.documents?.map((doc: any) => (
```

Comment:
I don't think **any** should be used if you can use the TYPE for this

Reply:
uki,i was negligent !!!!

Reply:
@nc-minh I have fixed the error, hope to be merged